sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesAction.cs

```diff
@@ -24,6 +24,20 @@ public RecognizePiiEntitiesAction()
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RecognizePiiEntitiesAction"/>
+        /// class based on the values of a <see cref="RecognizePiiEntitiesOptions"/>.
+        /// It sets the <see cref="ModelVersion"/>, <see cref="DisableServiceLogs"/>,
+        /// <see cref="DomainFilter"/>, and <see cref="CategoriesFilter"/> properties.
+        /// </summary>
+        public RecognizePiiEntitiesAction(RecognizePiiEntitiesOptions options)
+        {
+            ModelVersion = options.ModelVersion;
+            DisableServiceLogs = options.DisableServiceLogs;
+            DomainFilter = options.DomainFilter;
+            CategoriesFilter = options.CategoriesFilter ?? new List<PiiEntityCategory>(options.CategoriesFilter);
```

Comment:
If `options.CategoriesFilter` is null, is there any reason to pass it to the created list?

Reply:
It actually throws a `NullReferenceException` [according to the docs](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.-ctor?view=net-5.0#System_Collections_Generic_List_1__ctor_System_Collections_Generic_IEnumerable__0__).

Reply:
interesting.. I have no idea why I wrote that.
Thank you so much for catching that!